pkg/lintcontext/create_contexts.go

```diff
@@ -30,6 +42,7 @@ func CreateContexts(filesOrDirs ...string) ([]LintContext, error) {
 				continue
 			}
 			ctx := new()
+			ctx.customDecoder = options.CustomDecoder
```

Comment:
Can you pass `Options` into the `new` function and populate it there instead?

Also, this is not your code, but we should rename the function to `newCtx()` or something, since as of now, this will prevent the usage of the builtin `new` function in Go in this package. Can you do that, while we're here?

Reply:
Yup sounds good. Will do!